tests/test_notebooks.py

```diff
@@ -50,3 +50,8 @@ def test_06_information_geometry():
     @geomstats.tests.np_and_pytorch_only
     def test_07_k_mean_clustering_on_sphere():
         _exec_notebook('notebooks/06_k_mean_clustering_on_sphere.ipynb')
+
+    @staticmethod
+    @geomstats.tests.np_only
```

Comment:
It's enough for now to have it work with numpy only! Out of curiosity, did you see what fails with pytorch or tf?